weaver/wps_restapi/jobs/utils.py

```diff
@@ -78,6 +80,55 @@
 LOGGER = get_task_logger(__name__)
 
 
+def get_job_possible_output_formats(job):
+    """
+    Based on job output media-type, retrieve transformer possibilities (conversions)
+    """
+    outputs = []
+
+    for o in job.results:
+        outputs.append({
+            "output_id": o["identifier"],
+            "default_type": o["mimeType"],
+            "alternatives": [f for f in [fam for fam in transform.FAMILIES if
+                                         o["mimeType"] in fam and "/pdf" not in o["mimeType"]]]
+        })
+    return outputs
+
+
+def get_link(output_id, mime_type, url):
+    return {
+        "href": url + "/" + output_id + "?f=" + mime_type, "rel": "output:" + output_id,
+        "type": mime_type, "title": "Link to job " + output_id + " result in " + mime_type
```

Comment:
Reformat each field on its own line. Hard to read this way. Consider also using `f"{}"` stings.